src/components/command-bar/commands/search/unified-search/components/unified-hits-container/index.tsx

```diff
@@ -0,0 +1,104 @@
+// Libraries
+import { useMemo } from 'react'
+import { useHits } from 'react-instantsearch-hooks-web'
+// Global
+import Tabs, { Tab } from 'components/tabs'
+import { CommandBarDivider } from 'components/command-bar/components'
+import { CommandBarList } from 'components/command-bar/components'
+// Shared search
+import {
+	NoResultsMessage,
+	SuggestedPage,
+	SuggestedPages,
+	TabHeadingWithCount,
+} from '../../../components'
+// Unified search
+import { UnifiedHit } from '../unified-hit'
+import { gatherSearchTabsContent } from './helpers'
+// Types
+import type { ProductSlug } from 'types/products'
+import type { Hit } from 'instantsearch.js'
+import type { UnifiedSearchTabContent } from './helpers'
+// Styles
+import s from './unified-hits-container.module.css'
+
+/**
+ * Render search results from our unified index into content-type tabs.
+ *
+ * Note: this component needs to be used within an `InstantSearch` container
+ * imported from 'react-instantsearch-hooks-web'. That container provides
+ * the context from which `rawHits` are pulled.
+ */
+export function UnifiedHitsContainer({
```

Comment:
As with the `dialog-body` component in this PR, leaning heavily into splitting out helper functions and trying to document everything rather than going into a super heavily engineered refactor of the broader `CommandBar` etc.

Of note in this component, I tried to split out the "mostly business" logic of setting up the `tabData` vs the "mostly presentation" logic of rendering the tabs. Very open to iterating on this, in this PR and beyond 🙇‍♂️